docs/libcurl/curl_version_info.3

```diff
@@ -113,6 +113,9 @@ typedef struct {
   /* when 'age' is CURLVERSION_ELEVENTH or higher (>= 7.87.0), the members
      below exist */
   const char *const *feature_names; /* Feature names. */
+  /* when 'age' is CURLVERSION_TWELFTH or higher (>= 7.88.0), the members
+     below exist */
+  const char *ngtcp2_version; /* human readable string. */
```

Comment:
when ngtcp2 is used, so is nghttp3 which is a separate and independent library. Shouldn't that too be reported then?

Reply:
You are right. I did not see that. I'll try to update the code accordingly...

Reply:
I've implemented the same for HTTP3 now to be consistent. Build queue still runs but hopefully with a success (some always fails though - you are the expert....).